js/cdn/sample/src/constants/codeSnippets.ts

```diff
@@ -0,0 +1,214 @@
+
+  async function initMessenger() {
+    const messenger = await loadMessenger();
+
+    await messenger.initialize({
+      appId: "${APP_CONFIG.appId}",
+      aiAgentId: "${APP_CONFIG.aiAgentId}",
+      userSessionInfo: new messenger.ManualSessionInfo({
+        userId: "${APP_CONFIG.userId}",
+        authToken: "${APP_CONFIG.authToken}",
+        sessionHandler: {
+          onSessionTokenRequired: async (resolve) => {
+            resolve("${APP_CONFIG.authToken}");
+          },
+          onSessionClosed: () => {
+            console.log('Session closed');
+          },
+          onSessionError: (error) => {
+            console.error('Session error:', error);
+          },
+          onSessionRefreshed: () => {
+            console.log('Session refreshed')
+          }
+        }
+      })
+    });
+
+    console.log('Authenticated messenger ready');
+  }
+
+  initMessenger();
+</script>`,
+  },
+
+  context: {
+    title: 'With Context',
+    code: `<script type="module">
+  import { loadMessenger } from "https://aiagent.sendbird.com/orgs/default/index.js";
+
+  async function initMessenger() {
+    const messenger = await loadMessenger();
+
+    await messenger.initialize({
+      appId: "${APP_CONFIG.appId}",
+      aiAgentId: "${APP_CONFIG.aiAgentId}",
+      userSessionInfo: new messenger.AnonymousSessionInfo(),
+      context: {
+        userPreference: "technical",
+        customerTier: "premium",
+        supportLevel: "advanced"
+      }
+    });
+
+    console.log('Messenger with context ready');
+  }
+
+  initMessenger();
+</script>`,
+  },
+
+  runtime: {
+    title: 'Runtime Context Updates',
+    code: `<script type="module">
+  import { loadMessenger } from "https://aiagent.sendbird.com/orgs/default/index.js";
+  let messenger;
+
+  async function initMessenger() {
+    messenger = await loadMessenger();
+
+    await messenger.initialize({
+      appId: "${APP_CONFIG.appId}",
+      aiAgentId: "${APP_CONFIG.aiAgentId}",
+      userSessionInfo: new messenger.AnonymousSessionInfo()
+    });
+
+    console.log('Messenger ready');
+  }
+
+  async function updateContext(newContext) {
+    if (messenger) {
+      await messenger.patchContext(newContext);
+      console.log('Context updated:', newContext);
+    }
+  }
+
+  initMessenger().then(() => {
+    setTimeout(() => {
+      updateContext({ page: "checkout", cartItems: "3", cartValue: "100" });
+    }, 5000);
+  });
+</script>`,
+  },
+
+  localization: {
+    title: 'Multi-language',
+    code: `<script type="module">
+  import { loadMessenger } from "https://aiagent.sendbird.com/orgs/default/index.js";
+
+  async function initMessenger(lang = 'en', country = 'US') {
+    const messenger = await loadMessenger();
+
+    await messenger.initialize({
+      appId: "${APP_CONFIG.appId}",
+      aiAgentId: "${APP_CONFIG.aiAgentId}",
+      userSessionInfo: new messenger.AnonymousSessionInfo(),
+      language: lang,
+      countryCode: country
+    });
```

Comment:
initialize 가 비동기 함수가 아니라서 await 다 제거해도 될거같아용
```suggestion
    messenger.initialize({
      appId: "${APP_CONFIG.appId}",
      aiAgentId: "${APP_CONFIG.aiAgentId}",
      userSessionInfo: new messenger.AnonymousSessionInfo(),
      language: lang,
      countryCode: country
    });
```